publishing/submitting/README.md

```diff
@@ -0,0 +1,7 @@
+# Submitting your app to the App Warehouse
+
+To make your app discoverable by others, you're invited to publish it in the App Warehouse.
+
+Currently, the submitting apps to the App Warehouse is invitation-only.
+
+[Contact Starcounter](https://starcounter.com/about/) for more information.
```

Comment:
"App Warehouse" is actually more a business thing than a technical thing, isn't it? Won't the sales and marketing dept forward it to the responsible person?

@stefanedqvist any suggestions here how to contact Starcounter regarding the App Warehouse?